crates/apollo-compiler/CHANGELOG.md

```diff
@@ -55,9 +52,6 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
   A field argument or directive argument was incorrectly considered required
```

Comment:
can't comment on the right line as its not part of nthe diff, but missing a word "on"  in "relying on argument default" the line above here